### DISCUSSION TITLE
Additional widget (assigned issues)

### DISCUSSION THREAD

**Discussion by MiguelRipoll23** [community/community#3803] [Mobile] [answered]
A widget with the number of assigned issues and maybe a brief description would be nice!

Thanks.

**Comment by eliperkins** [comment, marked as answer]
@MiguelRipoll23 thanks for this feedback! We've got some ideas for more iOS widgets that we're exploring. I've added your notes to our internal tracking issue for this. Stay tuned!

**Reply by MiguelRipoll23** [reply]
That's great, thank you!
